From b1ebd3b0e4664c4aa8362bd8abb61861dff61849 Mon Sep 17 00:00:00 2001
From: Mike Christie <michael.christie@oracle.com>
Date: Thu, 18 Mar 2021 17:58:58 -0500
Subject: [PATCH] scsi: target: Fix htmldocs warning in target_submit_prep()
Git-commit: b1ebd3b0e4664c4aa8362bd8abb61861dff61849
Patch-mainline: v5.13-rc1
References: bsn#81
Modified-by-SEL: No


Fix warning:

drivers/target/target_core_transport.c:1661: WARNING: Block quote ends
without a blank line; unexpected unindent.

Link: https://lore.kernel.org/r/20210318225858.11863-1-michael.christie@oracle.com
Signed-off-by: Mike Christie <michael.christie@oracle.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Jun Yang <jun.yang@suse.com>
---
 drivers/target/target_core_transport.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/target/target_core_transport.c b/drivers/target/target_core_transport.c
index a75591c929c0..8fbfe75c5744 100644
--- a/drivers/target/target_core_transport.c
+++ b/drivers/target/target_core_transport.c
@@ -1659,6 +1659,7 @@ EXPORT_SYMBOL_GPL(target_init_cmd);
  * Returns:
  *	- less than zero to signal failure.
  *	- zero on success.
+ *
  * If failure is returned, lio will the callers queue_status to complete
  * the cmd.
  */
-- 
2.31.1

